seacrowd/sea_datasets/dengue_filipino/dengue_filipino.py

```diff
@@ -0,0 +1,132 @@
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@INPROCEEDINGS{8459963,
+      author={E. D. {Livelo} and C. {Cheng}},
+      booktitle={2018 IEEE International Conference on Agents (ICA)},
+      title={Intelligent Dengue Infoveillance Using Gated Recurrent Neural Learning and Cross-Label Frequencies},
+      year={2018},
+      volume={},
+      number={},
+      pages={2-7},
+      doi={10.1109/AGENTS.2018.8459963}}
+    }
+"""
+
+_LANGUAGES = ["fil", "tgl"]
+
+_DATASETNAME = "dengue_filipino"
+
+_DESCRIPTION = """\
+Benchmark dataset for low-resource multi-label classification, with 4,015 training, 500 testing, and 500 validation examples, each labeled as part of five classes. Each sample can be a part of multiple classes. Collected as tweets.
+"""
+
+_HOMEPAGE = "https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks"
+
+_LICENSE = Licenses.GPL_3_0.value
+
+_SUPPORTED_TASKS = []  # TODO: What's the appropriate task?
```

Comment:
Hi reviewers 👋 , need your input here. What's the appropriate task?

Looking through the existing single-text classification tasks, I don't think any of them fits.
```
# Single Text Classification
    ASPECT_BASED_SENTIMENT_ANALYSIS = "ABSA"
    EMOTION_CLASSIFICATION = "EC"
    HOAX_NEWS_CLASSIFICATION = "HNC"
    INTENT_CLASSIFICATION = "INT"
    LEGAL_CLASSIFICATION = "LC"
    RHETORIC_MODE_CLASSIFICATION = "RMC"
    SENTIMENT_ANALYSIS = "SA"
    TAX_COURT_VERDICT = "TACOS"
    TOPIC_MODELING = "TL"
```

Seems like we need to add a general text_multi_features task const?

Reply:
Hmmm, the task listed there has the schema defined for SEACrowd Dataloader. Although the multilabel features schema already exists, little to no tasks are mapped to the multilabel schema. Furthermore, since it's domain-specific classification (w/o linguistic context appropriate for this), it should lie under general classification, as you suggest.

I'll help on formulate this early next week (and link the upcoming PR to this one). Thanks for reporting, @ilhamfp!

Reply:
please help this issue

Reply:
Thanks! @sabilmakbar
Seems like I can use `DOMAIN_KNOWLEDGE_MULTICLASSIFICATION` now?

Reply:
yup, you can now if you have pulled the changes from `master`